shared/js/ui/views/mixins/open-options-page.es6.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Another shared method here since again, both main popup and hamburger menu need it